ansible/group_vars/all

```diff
@@ -62,6 +62,9 @@ runtimesManifest:
       default: true
       image:
         name: "swift3action"
+    - kind: "swift:3.1.1"
+      image:
+        name: "swift3.1.1action"
```

Comment:
had discussion with @rabbah I think we want to change the convention of runtime images.
Please change to `action-swift-v3.1.1`